### PR TITLE
Move user editable files out of the generated Xcode directory

### PR DESCRIPTION
**Pull request by chinmaygarde**

### CODE REVIEW

**Comment by chinmaygarde** [comment]
* The following files will remain unchanged across engine upgrades, the rest can be upgraded as the engine revisions are bumped up:
  * Info.plist
  * LaunchScreen.storyboard
  * Assets.xcassets (for application icons)
* Developers working on debugging the engine should still have a functional project harness
* Needs patches on the tooling front

**Comment by chinmaygarde** [comment]
cc @devoncarew

File: sky/build/LaunchScreen.storyboard [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="9059" systemVersion="14F27" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" launchScreen="YES" useTraitCollections="YES" initialViewController="01J-lp-oVM">
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="9531" systemVersion="15C50" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" launchScreen="YES" useTraitCollections="YES" initialViewController="01J-lp-oVM">
     <dependencies>
         <deployment identifier="iOS"/>
-        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="9049"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="9529"/>
```
Comment (devoncarew) [inline sky/build/LaunchScreen.storyboard]: Just curious, why do these numbers change?

**Comment by devoncarew** [comment]
lgtm, this is great

File: sky/build/LaunchScreen.storyboard (hunk 1 above)
Comment (chinmaygarde) [inline sky/build/LaunchScreen.storyboard]: Depends on which version of Xcode opened the file last

**Pull request merged**